WalletLibrary/Submodules/VerifiableCredential-SDK-iOS/VCNetworking/VCNetworking/operations/fetch/FetchPresentationRequestOperation.swift

```diff
@@ -30,7 +30,14 @@ class FetchPresentationRequestOperation: InternalNetworkOperation {
         self.correlationVector = cv
         
         /// sets value in order to get a signed version of the contract
-        self.urlRequest.addValue(Constants.VersionNumberHeaderValue,
-                                 forHTTPHeaderField: Constants.VersionNumberHeaderField)
+        if let value = urlSession.configuration.httpAdditionalHeaders?[Constants.VersionNumberHeaderField] {
```

Comment:
What does this do?

Reply:
This just concats the prefer header with one that exists globally. I think this could be refactored when we add support for Http Interceptor injection.